common/app/common/data.cljc

```diff
@@ -182,6 +182,20 @@
       (assoc m key (apply f found args))
       m)))
 
+(defn assoc-in-when
```

Comment:
Seems unused

Reply:
Yes, it's no longer used. We can let it there for completion or delete it, for me is ok.

Reply:
im prefer delete unless we really need it

Reply:
ok